example/airtable/elden-ring-map/inc/queries/class-airtable-elden-ring-list-locations-query.php

```diff
@@ -39,6 +39,6 @@ class AirtableEldenRingListLocationsQuery extends HttpQueryContext {
 	];
 
 	public function get_endpoint( array $input_variables ): string {
-		return $this->get_datasource()->get_endpoint( 'locations' ) . '?filterByFormula=FIND%28%27' . $input_variables['map_name'] . '%27%2C%20%7BMap%7D%29%3E0';
+		return $this->get_datasource()->get_endpoint() . '/tblc82R9msH4Yh6ZX?filterByFormula=FIND%28%27' . $input_variables['map_name'] . '%27%2C%20%7BMap%7D%29%3E0';
```

Comment:
Can we pull the table ID from the config? It would be ideal if getting the table ID here worked the same way with a hard coded data source as it does with a data source provided by the settings page.

Reply:
I feel pretty confident we can make smooth user experiences across both ways so I wouldn't fret relative to that angle. We will have excellent flexibility in UI/UX if we align our data constructs well down low with what they represent.

To me, a fundamental question here is what exactly _is_ an "Airtable data source"? What is the primary entity customers are connecting up to?

Is it the base? Is it the base+table?

Reply:
I think that's still somewhat of an open question and something we need feedback on. It requires more flexibility from our plugin, but I think ideally it can be all of the above. If the data source specifies a specific table, then it's essentially locked. If it doesn't, then the query config / UI has to specify the table for each case. So that could be in the query code, or perhaps we could build UI to support it in Gutenberg. But that last use case is down the road a bit.